gp2gp-translator/src/integrationTest/resources/application.yml

```diff
@@ -42,6 +42,7 @@ amqp:
 
 mhs:
   url: ${MHS_BASE_URL:http://localhost:8080}
+  maxRequestSize: ${MAX_REQUEST_SIZE:150000000}
```

Comment:
We'll want to document this variable within `OPERATING.md` and the `CHANGELOG`.

Reply:
@adrianclay thanks,  I've documented MAX_REQUEST_SIZE variable

Reply:
Okay, now I'm a bit more confused as the MHS outbound only really ever sends about 5MB messages. I thought the 150MB would have been more applicable to GP Connect Inbound responses. 🤔

Reply:
sorry, yes, you are right, I should have used the GP Connect Inbound instead of Outbound, will fix it.

Reply:
Similarly should this be inside of the `mhs` config element?